CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+Table of Contents
+=================
+* [Code of Conduct of Eclipse Che documentation project](#code-of-conduct-of-eclipse-che-documentation-project)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Attribution](#attribution)
+* [Pull Request Process](#pull-request-process)
+   * [Release branches](#release-branches)
```

Comment:
Can we add a short section on reviewing? I'd particularly like to call out that we need procedural PRs to get tested by QE before merging.